openslides/core/apps.py

```diff
@@ -46,6 +47,9 @@ def ready(self):
         post_permission_creation.connect(
             delete_django_app_permissions,
             dispatch_uid='delete_django_app_permissions')
+        post_permission_creation.connect(
```

Comment:
Please do not add migration process code here.

Reply:
Done. moved to migrations